apps/src/templates/WireframeButtons.jsx

```diff
@@ -106,11 +106,7 @@ let WireframeButtons = React.createClass({
   },
 
   renderNewProjectButton: function () {
-    // Unfortunately, isLegacyShare currently has the wrong value (is true for non-legacy artist
-    // and playlab). So instead, we check the first letter (after the /) in the path name of the
-    // url, as legacy shares all start with /c
-    // var isLegacyShare = this.props.isLegacyShare;
-    var isLegacyShare = window.location.pathname[1] === 'c';
```

Comment:
This code had noticed that something was wrong (the comment says `isLegacyShare currently has the wrong value`) - but the fix was hacky and narrow in scope.